src/platform/packages/shared/kbn-management/settings/components/field_input/code_editor.tsx

```diff
@@ -15,12 +15,13 @@
 //
 
 import React, { useCallback } from 'react';
-import { monaco, XJsonLang } from '@kbn/monaco';
```

Comment:
would it make sense to forbid importing monaco from @kbn/monaco and instead ask importing it from code_editor? (I don't know, just curious)

Reply:
I do agree, this is something worth exploring I created an issue for this.